e2e/cypress/integration/channel/scroll_channel_messages_spec.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Scroll channel`s messages in mobile view', () => {
+    before(() => {
+        cy.apiLogin('user-1');
+
+        // # resize browser to phone view
+        cy.viewport('iphone-6');
+
+        // # visit channel that has message on different days
+        cy.visit('ad-1/channels/minima-3');
```

Comment:
Would be better to use new channel so we can test 3 cases: Today, Yesterday and 2 days ago.
```suggestion
let newChannel;
...
        // # Create and visit new channel
        cy.createAndVisitNewChannel().then((channel) => {
            newChannel = channel;
        });
```

Reply:
Then post into that channel several messages, like 5 messages each for Today, Yesterday and 2 days ago.  Can be done like
```javascript
// # Post other day messages
const otherDay = Cypress.moment().subtract(2, 'days');
for (let i = 0; i < 5; i++) {
    postAMessage(`Hello \n from \n other \n day \n - ${i}`, newChannel.id, otherDay.valueOf() + i);
}

// note: need to "cy.reload()" after posting all messages to get the ordered post list
```